### DISCUSSION TITLE
Problem installing laravel 9.5.2 with curl command (docker) on windows WSL2 and Mac m1

### DISCUSSION THREAD

**Discussion by Glifaus** [laravel/framework#45889] [General] [answered]
Hi! Im David and new here.

I tried to start a new project, easy with the curl command but dont work.
In windows 11 with wsl2 and macOs 13.2 (air m1) show the same log.

`Invalid services [mailhog].
no configuration file provided: not found
no configuration file provided: not found
no configuration file provided: not found
no configuration file provided: not found

Please provide your password so we can make some final adjustments to your application's permissions.`

Can be this a problem of the image or sail? This must work withouth problem... 

also i try to install sail and run sail up again. Mailhog is not in the services list:
` Which services would you like to install? [mysql]:
  [0] mysql
  [1] pgsql
  [2] mariadb
  [3] redis
  [4] memcached
  [5] meilisearch
  [6] minio
  [7] mailpit
  [8] selenium`

I dont know really if its my fault or a issue ( wich i think) that have to be resolved.

Any ideas? 
Thanks!!

**Comment by Don-Guille** [comment, marked as answer]
Hi, I'm Guillermo.
I am also trying to create a new project and it shows me the same error on linux (Manjaro)
```
Invalid services [mailhog].
no configuration file provided: not found
no configuration file provided: not found
no configuration file provided: not found
no configuration file provided: not found
```

**Reply by Glifaus** [reply]
Hi again!
Seems that it's fixed. Now laravel.build script use mailpit and works again out of the box.

Bash script has changed:

> docker run --rm \
>     --pull=always \
>     -v "$(pwd)":/opt \
>     -w /opt \
>     laravelsail/php82-composer:latest \
>     bash -c "laravel new test-mailpit && cd test-mailpit && php ./artisan sail:install --with=mysql,redis,meilisearch,mailpit,selenium "
> 
> cd test-mailpit
> 
> ./vendor/bin/sail pull mysql redis meilisearch mailpit selenium
> ./vendor/bin/sail build

I try to install and now all is succesfull.
Maybe a error or late change with the new version that came out yesterday.

**Reply by Don-Guille** [reply]
thanks!

**Comment by luisferfranco** [comment]
I'm having exactly the same problem on Windows 11. The script I get is this (it looks exactly as what you posted), and yet isn't working.

```
docker run --rm \
    --pull=always \
    -v "$(pwd)":/opt \
    -w /opt \
    laravelsail/php82-composer:latest \
    bash -c "laravel new example-app && cd example-app && php ./artisan sail:install --with=mysql,redis,meilisearch,mailpit,selenium "

cd example-appcd

# Allow build with no additional services..
if [ "mysql redis meilisearch mailpit selenium" == "none" ]; then
    ./vendor/bin/sail build
else
    ./vendor/bin/sail pull mysql redis meilisearch mailpit selenium
    ./vendor/bin/sail build
fi
```
